app/controllers/BaseController.java

```diff
@@ -49,19 +49,15 @@
 
     private static ObjectMapper mapper = BridgeObjectMapper.get();
 
-    protected AuthenticationService authenticationService;
-    protected StudyService studyService;
-    protected CacheProvider cacheProvider;
-    protected BridgeConfig bridgeConfig;
+    private BridgeConfig bridgeConfig;
+    private CacheProvider cacheProvider;
 
-    @Autowired
-    public void setStudyService(StudyService studyService) {
-        this.studyService = studyService;
-    }
+    StudyService studyService;
```

Comment:
"protected" has more visibility than package scope.  Members in the same package can access protected fields even if there is not inheritance.  I'd like to reduce it to private scope but that involves too many places to change.  In general, I'm very much against any field that is not private.  But let's reduce it to package scope first.